### PR TITLE
fixed bug in console

### PR DESCRIPTION
**Pull request by xkevin190**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -24,7 +24,7 @@ prost = "0.6"
 # Not available on WASM
 [target.'cfg(not(target_os = "unknown"))'.dependencies]
 miniupnpc = { version = "0.1", path = "miniupnpc" }
-
+base64 = "0.13"
```
Comment (jeandudey) [inline Cargo.toml]: Move to the upper dependency block as this is for dependencies that can't be compiled to WebAssembly.

**Review by luisan00 — APPROVED** [review]
💯

File: locha-p2pd/src/main.rs [hunk 2]
```diff
@@ -67,10 +67,10 @@ pub fn deserialize_message(buf: &[u8]) -> String {
         r#type: content.type_message,
         msg: Msg {
             text: content.text,
-            file: if content.file.is_empty() {
-                None
+            file: if content.file.len() == 0 {
```
Comment (jeandudey) [inline locha-p2pd/src/main.rs]:
```suggestion
            file: if content.file.is_empty() {
```

File: locha-p2pd/src/main.rs [hunk 3]
```diff
@@ -118,7 +118,7 @@ pub fn serialize_message(contents: String) -> Vec<u8> {
     message.timestamp = datetime.as_secs();
     message.type_message = 1;
     message.text = text_message;
-    message.file = String::new();
+    message.file = base64::decode(String::new()).expect("base64 could not be decoded");
```
Comment (jeandudey) [inline locha-p2pd/src/main.rs]:
```suggestion
    message.file = Vec::new();
```

**Review by jeandudey — APPROVED** [review]

**Comment by jeandudey** [comment]
CI not happy :-(, run: `cargo fmt --all`

**Pull request merged**